static/spec/openapi.json

```diff
@@ -371,7 +371,7 @@
           "partial",
           "pending",
           "running",
-          "complete",
+          "completed_at",
```

Comment:
This references the status of export being `complete`, which is different from the time it was completed `completed_at`.